docs/source/common/lightning_module.rst

```diff
@@ -731,6 +731,112 @@ Child Modules
 
 .. include:: ../common/child_modules.rst
 
+-----------
+
+****************
+Hooks Call Order
```

Comment:
I'm still not convinced that this is that useful because one has to click through these menus to find the hooks.

Ideally, we would add links in the pseudocode we have...